server/offenceCodeDecisions/Commited.ts

```diff
@@ -0,0 +1,153 @@
+import Question from './Question'
+import Code from './Code'
+
+// Codes
+const todo = new Code('TODO')
+
+function decision(question: Question | string) {
+  return new Decision(question)
+}
+
+export default new Decision()
+  .title('What type of offence did {OFFENDED_FIRST_NAME} {OFFENDER_LAST_NAME} commit?')
+  .child(
+    decision('Assault, fighting, or endangering the health or personal safety of others')
+      .title('What did the incident involve?')
+      .child(
+        decision('Assaulting someone')
+          .title('Who was assaulted?')
+          .url('incident-involved/1')
+          .child(
+            decision('Another prisoner')
+              .title('Was the incident a racially aggravated assault?')
+              .child(decision('Yes').code(todo))
+              .child(decision('No').code(todo))
+          )
+          .child(
+            decision('A prison officer')
+              .title('Was the incident a racially aggravated assault?')
+              .child(decision('Yes').code(todo))
+              .child(decision('No').code(todo))
+          )
+          .child(
+            decision('A member of staff who is not a prison officer')
+              .title('Was the incident a racially aggravated assault?')
+              .child(decision('Yes').code(todo))
+              .child(decision('No').code(todo))
+          )
+          .child(
+            decision('Another person not listed above')
+              .title('Was the incident a racially aggravated assault?')
+              .child(decision('Yes').code(todo))
+              .child(decision('No').code(todo))
+          )
+      )
+      .child(decision('Fighting with someone').code(todo))
+      .child(decision('Endangering the health or personal safety of someone').code(todo))
+  )
+  .child(
+    decision('Escape or failure to comply with temporary release conditions')
+      .title('What did the incident involve?')
+      .child(decision('Escaping').code(todo))
+      .child(decision('Absconding from either prison or legal custody').code(todo))
+      .child(decision('Failing to comply with any conditions of a temporary release').code(todo))
+      .child(decision('Failing to return from their temporary release').code(todo))
+  )
+  .child(
+    decision('Possession of unauthorised articles, or drugs or alcohol related')
+      .title('What did the incident involve?')
+      .child(
+        decision('Possession of an unauthorised article')
+          .title('What happened?')
+          .child(
+            decision('Has unauthorised article in possession')
+              .title('Did they have a greater amount than they are allowed to have?')
+              .child(decision('Yes').code(todo))
+              .child(decision('No').code(todo))
+          )
+          .child(
+            decision('Sells or gives an unauthorised article to another person')
+              .title('Was the article only for {OFFENDED_FIRST_NAME} {OFFENDER_LAST_NAME}`s personal user')
```

Comment:
We have to do token replacement in some of the questions - its pretty simple matching

Reply:
is there a reason why you don't use PlaceholderText (or in fact, is PlaceholderText required at all)?

Reply:
So this was tied up with not being able to get enums working - however I think I've fixed that now and we use the enums in the titles here.